config/config.go

```diff
@@ -606,6 +606,8 @@ type ScrapeConfig struct {
 	// More than this many buckets in a native histogram will cause the scrape to
 	// fail.
```

Comment:
We forgot to update the doc comment here in the previous PR. Should now read something like : "More than this many buckets in a native histogram will automatically merge buckets to stay within the limit."